easybuild/easyblocks/g/gromacs.py

```diff
@@ -97,7 +102,10 @@ def configure_step(self):
                 for libname in ['BLAS', 'LAPACK']:
                     lib_dir = os.getenv('%s_LIB_DIR' % libname)
                     libs = os.getenv('LIB%s' % libname)
-                    self.cfg.update('configopts', '-DGMX_%s_USER="-L%s %s"' % (libname, lib_dir, libs))
+                    if self.toolchain.toolchain_family() == toolchain.CRAYPE:
+                        self.cfg.update('configopts', '-DGMX_%s_USER="%s/libsci_gnu_mpi_mp.a"' % (libname, lib_dir))
```

Comment:
@gppezzi can you change this like I did in https://github.com/hpcugent/easybuild-easyblocks/pull/960/files#diff-a02b2451ce7221a206c1f64ba6758d33R158 (only the Cray-specific part in this `if` block), then I can merge this PR (and sync #960 after the merge)